app/graphql/types/profile.rb

```diff
@@ -44,6 +44,14 @@ class Types::Profile < Types::BaseObject
     null: true,
     description: 'The message this user has submitted for the Hall of Fame'
 
+  field :site_links, Types::SiteLink.connection_type,
+    null: true,
+    description: 'Links related to a profile.'
```

Comment:
1. Links to the user on other (social media) sites

2. The user's accounts on other (social media) sites